### PR TITLE
test: correct false-positive helgrind warnings

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

File: qa/workunits/rbd/test_librbd.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 if [ -n "${VALGRIND}" ]; then
   valgrind --tool=${VALGRIND} --suppressions=${TESTDIR}/valgrind.supp \
-    ceph_test_librbd
+    --error-limit=no --error-exitcode=1 ${VALGRIND_ARGUMENTS} ceph_test_librbd
```
Comment (jdurgin) [inline qa/workunits/rbd/test_librbd.sh]: maybe make it a large exit code to distinguish it from non-valgrind failures easily in job summaries

Comment (dillaman) [inline qa/workunits/rbd/test_librbd.sh]: Suggestion?

Comment (jdurgin) [inline qa/workunits/rbd/test_librbd.sh]: perhaps 140 since it's higher than any errno

**Comment by trociny** [comment]
lgtm

**Comment by dillaman** [comment]
failures still present: http://pulpito.ceph.com/yuriw-2016-06-07_18:48:31-rbd-wip-yuri-testing_JUN_7_2016---basic-smithi/

**Pull request closed without merging**